equinox-shipment/Domain/Container.fs

```diff
@@ -0,0 +1,56 @@
+module Container
+
+type Container = { id: string; shipmentIds: string[]; finalized: bool }
```

Comment:
`id : string` is about to become the official layout standard!

Reply:
suggest `ContainerState` to emphasize it's state (and goes in events)
suggest putting comment aobve indicating it needs to be serializable
replace shipmentIds with Fsharp.UMX

Reply:
Sure, `id` and `shipmentIds` aren't needed so I'll remove them.